.gitignore

```diff
@@ -4,6 +4,7 @@
 *_Spoiler.txt
 *.bmbp
 *.apbp
+*.apcivvi
```

Comment:
Oh, I just realised something.

You should add your patch extension to inno_setup.iss

That way, it gets file association, meaning that people can just double click .apcivvi files to launch the Civ6 client.

Reply:
Ah nice, I'll add that now!

Reply:
@NewSoupVi I just updated but am unsure if I need to have a different path to the client since the client is nested in a world folder? Let me know if you get a chance to take a look 👍

Reply:
(answered in discord, resolving)